appveyor.yml

```diff
@@ -25,33 +25,63 @@ cache:
 
 environment:
   global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\bin\\admin\\run_with_env.cmd"
+
     TRIAL_REPORTER: text
 
   # Only use pre-installed Python versions, which Appveyor may upgrade to
   # a later point release.
   # See: http://www.appveyor.com/docs/installed-software#python
   matrix:
-    - PYTHON: C:\Python27-x64
+    - PYTHON_HOME: C:\\Python27-x64
+      PYTHON_VERSION: "2.7"
+      PYTHON_ARCH: "64"
       TOXENV: py27-coverage-windows,codecov-publish
+    #- PYTHON_HOME: "C:\\PYTHON35-x64"
```

Comment:
I wanted to leave those lines in there, but commented out, so
that I can work on a branch with those lines uncommented and gradually get a working Windows Python 3 appveyor build, and also to allow other folks to help the Windows Python 3 effort by uncommenting those lines in their private branches.

Right now I have been chasing all the appveyor changes in a branch in a separate repository and it has been painful.